definitions/features/foreman_proxy.rb

```diff
@@ -226,6 +233,18 @@ def lookup_using_dhcp_yml
     end
   end
 
+  def lookup_tftp_root_directory
+    tftp_yml_path = lookup_into(FOREMAN_PROXY_TFTP_YML_PATHS)
+    raise "Couldn't find tftp.yml file under foreman-proxy" unless tftp_yml_path
+
+    configs_from_tftp_yml = yaml_load(tftp_yml_path)
+    if configs_from_tftp_yml.key?(:tftproot)
+      return configs_from_tftp_yml[:tftproot]
+    else
+      raise "Couldn't find TFTP Configurations in #{tftp_yml_path}"
```

Comment:
Do we need to raise error here ? I mean is it valid condition tftp feature is enabled but not configured ?

Reply:
Confirmed that in any case(external/internal TFTP) ":tftproot" will be configured, so removed the above condition.